api/models/ExportedEAppData.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Model ExportedEAppData.js
+ * @module Model ExportedEAppData
+ */
+
+module.exports = {
+    attributes: {
+        application_id: {
+            type: Sequelize.INTEGER(),
+        },
+        applicationType: {
+            type: Sequelize.STRING(),
+        },
```

Comment:
As in using camel case instead of snake case? Erm, no it's not too late to change, would be a bit of a faff but should be doable. Although my inspiration was from this file:

https://github.com/UKForeignOffice/loi-application-service/blob/develop/api/models/ExportedApplicationData.js#L12

They are pretty much doing the same thing except one is for electronic apps and one is for paper apps.